### PR TITLE
gibo: 1.0.6 -> 2.2.4

### PR DESCRIPTION
**Pull request by Amir-Ahmad**
###### Motivation for this change
Update from 1.0.6 to 2.2.4

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118380` at 10d036c9 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>gibo</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/tools/misc/gibo/default.nix:28:3:
```
   |
28 |   meta = {
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/tools/misc/gibo/default.nix:16:3:
```
   |
16 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/tools/misc/gibo/default.nix [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
     repo = "gibo";
     rev = version;
-    sha256 = "07j3sv9ar9l074krajw8nfmsfmdp836irsbd053dbqk2v880gfm6";
+    sha256 = "0d3596yfyic6sarny23aw4yrb6gr7adpiw5cxx20fqj7xpw72m7a";
   };
 
   phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
 
   installPhase = ''
     mkdir -p $out/bin $out/share/bash-completion/completions
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/gibo/default.nix:17]:
```suggestion
    mkdir -p $out/bin
```

File: pkgs/tools/misc/gibo/default.nix [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
     cp gibo $out/bin
-    cp gibo-completion.bash $out/share/bash-completion/completions
+    cp shell-completions/gibo-completion.bash $out/share/bash-completion/completions
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/gibo/default.nix:19]:
```suggestion
    installShellCompletion --bash shell-completions/gibo-completion.bash
```

File: pkgs/tools/misc/gibo/default.nix [hunk 3]
```diff
@@ -2,21 +2,21 @@
...
 
   src = fetchFromGitHub {
     owner = "simonwhitaker";
     repo = "gibo";
     rev = version;
-    sha256 = "07j3sv9ar9l074krajw8nfmsfmdp836irsbd053dbqk2v880gfm6";
+    sha256 = "0d3596yfyic6sarny23aw4yrb6gr7adpiw5cxx20fqj7xpw72m7a";
   };
 
   phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/gibo/default.nix:14]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Closing due to inactivity from author.

**Pull request closed without merging**